src/whylogs/app/logger.py

```diff
@@ -75,45 +80,82 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
+
+    @property
+    def profile(self,):
+        """
+        :return: the last backing dataset profile
+        :rtype: DatasetProfile
+        """
+        return self._profiles[-1]["full_profile"]
+
     @property
-    def profile(self):
+    def segmented_profiles(self,)->DatasetProfile:
         """
         :return: the last backing dataset profile
         :rtype: DatasetProfile
         """
-        return self._profiles[-1]
+        
+        return self._profiles[-1]["segmented_profiles"]
```

Comment:
the name reads `segmented_profiles ` but we're getting a single profile back? The logic doesn't make sense to me here

Reply:
you get a `Dict[str,DatasetProfile]`, so you have acess to each segment. i will update the documentation.

Reply:
Done